src/common/components/modal/modal.module.scss

```diff
@@ -32,35 +36,42 @@ $modalPadding: 2rem;
   justify-content: center;
 }
 
+.closeButton {
+  float: right;
+  @include respond-below(md) {
+    z-index: 1;
+    position: relative;
+    text-align: right;
```

Comment:
this is not needed no more

Reply:
With the current implementation (float right) you need text-align to make it stay to the right. 
I'll change it to flex.

Reply:
fair enough